src/components/LanguageBar/LanguageBar.tsx

```diff
@@ -55,53 +55,57 @@ export const LanguageBar: FunctionComponent<LanguageBarProps> = ({
   setSelectedLanguage,
   showDisabled = false,
 }) => {
+  const displayable = showDisabled ? LANGUAGES : targetLanguages;
   return (
-    <Flex data-testid="language-bar">
-      {(showDisabled ? LANGUAGES : targetLanguages).map(
-        (language: Language) => {
-          const isDisabled = !targetLanguages.includes(language);
-          const isSelected = language === selectedLanguage;
+    <Grid
+      data-testid="language-bar"
+      gap="2"
+      gridTemplateColumns={`repeat(${displayable.length}, 1fr)`}
```

Comment:
The advantage of using grid is that it forces each item to have the exact same width